CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 ## Unreleased
 
+## 0.2.7
+
 - Added new endpoint for customer search
 - Added new endpoints for draft orders
+- Added the ShopifyAPI namespace to the graphQL schema
```

Comment:
Should clarify a bit further:

- Added `ShopifyAPI` namespace prefix to `GraphQL` modules.

Reply:
done - will squash commits before merging